packages/compass-saved-aggregations-queries/test/mock.ts

```diff
@@ -85,10 +88,10 @@ export function createCompassQueryHistoryMock(queries: Query[]): {
 
 export function createProxyquireMockForQueriesAndAggregationsPlugins(
   pipelines: StoredPipeline[],
-  queries: Query[]
+  queries: FavoriteQuery[]
 ): unknown {
   return {
-    '@mongodb-js/compass-query-history': {
+    '@mongodb-js/compass-query-bar': {
       ...createCompassQueryHistoryMock(queries),
```

Comment:
I am not sure if we still require this complicated setup for this plugin, but I'll check it and if removable, create a follow up.